parsl/__init__.py

```diff
@@ -46,6 +46,7 @@
     'bash_app',
     'python_app',
     'join_app',
+    'watch_app',
```

Comment:
I don't see where this is defined

Reply:
This was a left over, it has been removed.